automation.py

```diff
@@ -36,15 +41,20 @@ def automate_passovers():
     be automatically run again during the next pass.
     """
     passover = PassoverList()
+    housekeeping = HousekeepingLogList()
 
     # the automation will also handle queuing passover times
     passovers = passover.get(local_args={'limit': 1, 'next': 'true'})
+    last_pass = passover.get(local_args={'most-recent': 'true'})[0]['data']['most_recent_passover']
+
+    message = 'An Ex-Alta 2 passover is beginning now! The timestamp for this passover is {0}'.format(last_pass['timestamp'])
+    send_slack_notifs(message)
```

Comment:
This may need to be moved in `automate_communication()`. From what I've gathers from how `automate.sh` is scheduled to run, when a passover starts, `automate_communication()` will get called first then 60 seconds later `automate_passovers()` will get called. 

If we move this block to `automate_communication()` (and maybe modify the function to have an timestamp parameter so we don't need to instantiate a passover list), then the slack notification should be sent out at the very beginning of the passover.

If it already works correctly and messages are being sent at the beginning of a passover, then disregard this message :)

Reply:
Updated!